src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -878,6 +879,17 @@ public function providePathsForMakePathRelative()
             array('C:/aa/bb/../../cc', 'C:/aa/../dd/..', 'cc/'),
             array('C:/../aa/bb/cc', 'C:/aa/dd/..', 'bb/cc/'),
             array('C:/../../aa/../bb/cc', 'C:/aa/dd/..', '../bb/cc/'),
+            array('aa/bb', 'aa/cc', '../bb/'),
```

Comment:
This is not something that is supported. The docblock states that the paths have to be absolute.